### ISSUE TITLE
Do not use native browser alert()

### ISSUE THREAD

**Issue by wwalc** [cksource/ckeditor-plugin-a11ychecker#100]
Since a11y checker is heavily using its own dialogs I believe that when no issues are found, for consistency CKEditor dialog should be used to notify about this as well:

![image](https://cloud.githubusercontent.com/assets/545175/6128000/720f90a2-b131-11e4-9e05-14297230644e.png)

Not sure what others think about this, so marking it as a question.

**Comment by mlewand** [comment]
A duplicate of this enhancement has been requested in #225.
